foundry.toml

```diff
@@ -0,0 +1,21 @@
+[default]
+solc_version = "0.8.7"
```

Comment:
Something's off, the CI says that it compiles contracts using Solidity 0.8.12, but THB I don't understand why?

Reply:
I just noticed that this also happens on my local machine. This is probably foundry bug - I will investigate that more.

Reply:
Seems like setting `FOUNDRY_SOLC_VERSION` does not work either.